ts/screens/wallet/payment/TransactionSummaryScreen.tsx

```diff
@@ -79,6 +92,66 @@ export type TransactionSummaryScreenNavigationParams = Readonly<{
   messageId?: string;
 }>;
 
+export type TransactionSummaryErrorContent = PayloadForAction<
+  | typeof paymentVerifica["failure"]
+  | typeof paymentAttiva["failure"]
+  | typeof paymentIdPolling["failure"]
+>;
+
+export type TransactionSummaryError = O.Option<TransactionSummaryErrorContent>;
+
+const styles = StyleSheet.create({
+  container: {
+    paddingHorizontal: customVariables.contentPadding
+  }
+});
+
+const renderFooter = (
+  isLoading: boolean,
+  error: TransactionSummaryError,
+  continueWithPayment: () => void,
+  help: () => void
+) => {
+  if (O.isSome(error)) {
+    const errorOrUndefined = O.toUndefined(error);
+    const errorType = getV2ErrorMainType(errorOrUndefined as DetailV2Keys);
+    switch (errorType) {
+      case "TECHNICAL":
+      case "DATA":
+      case "UNCOVERED":
+        return (
+          <ButtonSolid
+            onPress={help}
+            fullWidth
+            accessibilityLabel={I18n.t("payment.details.info.buttons.help")}
+            label={I18n.t("payment.details.info.buttons.help")}
+          />
+        );
+      // There's a strange case where error is 'some' but
+      // its value is undefined (e.g. network error).
+      // In this case we fallback to the 'continue' CTA
+      // so that the user can eventually retry.
+      case undefined:
+        break;
+      default:
+        return <></>;
+    }
+  }
+  if (isLoading) {
+    // TODO: Add a ButtonOutline with a loading spinner as soon as it's available by DS (https://pagopa.atlassian.net/browse/IOBP-302)
+    return <></>;
+  }
```

Comment:
IMHO, it doesn't make much sense to merge a PR without this component. Currently, when the transaction is being confirmed, the button disappears and it's not a great UX.
If this redesign isn't urgent I think we should wait.
What do you think?

Reply:
We need to know when the button with loading spinner inside will be available into design system, anyway for this question I suggest to mention @dmnplb and @thisisjp that could give us more information about the urgency

Reply:
@mastro993 Meanwhile, in order to await the new `isLoading` prop to the `ButtonSolid`, with this commit 17e1f0cdcfac43b5714d3a671226a5d0a727c36a I provided to add a disabled button while loading

Reply:
That makes sense! Let’s add the loading state in the backlog